### ISSUE TITLE
System shutdown itself after library scan jellyfin/jellyfin:10.8.0-alpha5-amd64 [Issue]: 

### ISSUE THREAD

**Issue by 0TTA** [jellyfin/jellyfin#7124]
### Please describe your bug

It works without problems, but when I manually scan the library, it scans it, and then the container shutdown itself. It works fine, though, after I manually rerun the container.

### Jellyfin Version

Other

### if other:

10.8.0-alpha5

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wolonggl** [comment]
If you scan a large number of library video files, the ubuntu version has the same problem. The scanning process freezes and the Web page does not respond.

**Comment by 0TTA** [comment]
Yeah, I'm on unRAID. I had some issues too with HDR transcoding, so, I pull back to alpha3.

**Comment by NotExpectedYet** [comment]
Have this on UnRaid myself. Interface becomes unresponsive on a scan.

**Comment by 0TTA** [comment]
Fixed on jellyfin/jellyfin:20220107.9-unstable
